Generals/Code/Tools/matchbot/generals.cpp

```diff
@@ -626,7 +626,7 @@ void GeneralsMatcher::checkMatchesInUserMap(UserMap& userMap, int ladderID, int
 									{
 
 										i5 = i4;
-										for (++i5; i5 != userMap.end(); ++i3)
+										for (++i5; i5 != userMap.end(); ++i5)
```

Comment:
Are you sure that is a typo ? Can you reproduce a bug on that ?
i5 is already incremented in this loop, I'm pretty we should not increment it twice because it can miss the condition `i5 != userMap.end()`